File: app/src/main/java/ru/vincetti/vimoney/data/models/json/CurrencyItem.java
package ru.vincetti.vimoney.data.models.json;

import com.google.gson.annotations.SerializedName;

public class CurrencyItem{

	@SerializedName("code")
	private int code;

	@SerializedName("name")
	private String name;

	private String symbol;

	public int getCode(){
		return code;
	}

	public String getName(){
		return name;
	}

	public String getSymbol() {
		return symbol;
	}
}

File: app/src/main/java/ru/vincetti/vimoney/data/models/json/CurrencyItem.kt
package ru.vincetti.vimoney.data.models.json

import com.google.gson.annotations.SerializedName

data class CurrencyItem(

        @SerializedName("code")
        val code: Int,

        @SerializedName("name")
        val name: String,

        val symbol: String
)